docs/latest/user/rate-limit.md

```diff
@@ -591,6 +601,11 @@ data:
           replicas: 1
           container:
             image: envoyproxy/ratelimit:master
+            env:
+            - name: env_a
```

Comment:
this example would be more useful, if some existing env was used e.g. `CACHE_KEY_PREFIX`
https://github.com/envoyproxy/ratelimit#redis